security/cmd/istio_ca/main.go

```diff
@@ -252,11 +379,38 @@ func createCA(core corev1.SecretsGetter) ca.CertificateAuthority {
 	}
 
 	caOpts.LivenessProbeOptions = opts.LivenessProbeOptions
+	caOpts.ProbeCheckInterval = opts.probeCheckInterval
 
 	istioCA, err := ca.NewIstioCA(caOpts)
 	if err != nil {
 		log.Errorf("Failed to create an Istio CA (error: %v)", err)
 	}
+
+	if opts.LivenessProbeOptions.IsValid() {
```

Comment:
Could you move this block to a separate file in security/pkg/probe/?

Reply:
Done